.github/workflows/code-qa.yml

```diff
@@ -25,3 +25,21 @@ jobs:
 
       - name: Compile TypeScript
         run: npm run compile
+
```

Comment:
On line 11 I would rename `code-qa` to `compile`, since all the jobs in this workflow, including `unit-test` are part of code quality assurance

Reply:
![image](https://github.com/user-attachments/assets/2a039338-2663-49d1-befa-72302f4bf00e)